activity-tracker-root/activity-tracker/src/main/java/io/fourfinance/activity_tracker/activity/TrackUserActivity.java

```diff
@@ -0,0 +1,13 @@
+package io.fourfinance.activity_tracker.activity;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface TrackUserActivity {
+
+    String value() default "empty";
```

Comment:
Don't think that default value here is good. Each usage of this annotation should lead to provide name. 
Also you can give this more meaningful name e.g. metricName